source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -36,6 +36,19 @@ content: |
              if (err) throw err;
              localMasterKey = data;
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 4
+
+           from bson import binary
+           
+           const path = "./master-key.txt";
```

Comment:
Does `const` have significance in Python? I think maybe you meant:
```
path = "./master-key.txt";
```

Reply:
It doesn't have significance in py. Fixed:

- Removed the 'const'